interfaces/api/stats/services/stats_service.py

```
@@ -27,20 +32,91 @@ def __init__(self, repository: StatsRepository):
+                COALESCE(SUM(CASE WHEN c.content IS NOT NULL
+                                  AND c.content != '' THEN 1 ELSE 0 END), 0) AS chapters_with_content
+            FROM novels n
+            LEFT JOIN chapters c ON n.id = c.novel_id
+        """)
+
+        total_books = row["total_books"] if row else 0
+        total_chapters = row["total_chapters"] if row else 0
+        total_characters = row["total_characters"] if row else 0
+        chapters_with_content = row["chapters_with_content"] if row else 0
+
+        # 估算总字数：中文为主时 ≈ 字符数；有英文时需加回英文单词
```

Ruff (0.15.13)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)